src/main/java/com/gd/clinic/entity/Treatment.java

```diff
@@ -0,0 +1,45 @@
+package com.gd.clinic.entity;
+
+import com.gd.clinic.model.NewTreatmentDto;
+import lombok.*;
+import org.hibernate.Hibernate;
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Type;
+
+import javax.persistence.*;
+import java.util.Objects;
+import java.util.UUID;
+
+@Getter
+@Setter
+@ToString
+@Entity
+@Table(name = "treatments")
+public class Treatment {
```

Comment:
seems there should be relation to Prescription, isn't it?

Reply:
This table is meant to be used as a manageable catalogue for treatments(procedure and medicine listing only), it does not account for a relation with prescriptions, since many prescriptions for different patients may use the same treatment. Prescriptions, on the other hand do have this sort of relationship with patients.